### ISSUE TITLE
ms update did not install dependency

### ISSUE THREAD

**Issue by mmundy42** [ModelSEED/ModelSEED#167]
I used the "ms update" command to update my developer package install to the latest version and the YAML:XS package dependency was not installed.  The model command failed because the package wasn't installed on my system.  I installed the package manually and then the command worked.  Should the package have been installed by the update command?

While debugging I changed the update command to verify the developer package was being used.  Here's a patch.

diff --git a/lib/ModelSEED/App/mseed/Command/update.pm b/lib/ModelSEED/App/mseed/Command/update.pm
index 00e77ee..d20ee9f 100644
--- a/lib/ModelSEED/App/mseed/Command/update.pm
+++ b/lib/ModelSEED/App/mseed/Command/update.pm
@@ -72,7 +72,7 @@ sub nix_update_dev {
         }
     }
-    print "Beginning update...\n";
-    print "Beginning update of developer package ...\n";
  
   # should do checking for MS and MFA directories in perl, warn if not there.

@@ -127,6 +127,7 @@ sub nix_update_dev {

 sub nix_update_user {
     my ($self) = @_;
-    print "Beginning update of user package ...\n";
  
   # check if new tarball is available, if yes, download and uncompress
   # then system commands to compile

**Comment by cshenry** [comment]
Yes. This dependency should be added to the "Build.pl". Then everything should work. I'll handle this.
